src/main/webapp/app/shared/links/Linkout.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { CSSProperties } from 'react';
+
+export const Linkout: React.FunctionComponent<{
+  to: string;
+  className?: string;
+  style?: CSSProperties;
+  addHttpsProtocol?: boolean;
+}> = props => {
+  let updatedLink = props.to;
+  if (props.addHttpsProtocol === undefined || props.addHttpsProtocol) {
```

Comment:
Are we defaulting to true here? In that case, think we should use default assignment